website/src/stories/Header.stories.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { SessionContext } from 'next-auth/react';
+
+
+import { Header } from '../components/Header';
```

Comment:
Is it possible to import from `src/components/Header`? If not, that's okay

Reply:
moved them into the same folder, think thats inline with #99